Gruntfile.js

```diff
@@ -29,6 +29,14 @@ module.exports = function (grunt) {
       }
     }()),
 
+    i18nextract: {
+      default_options: {
+        src: [ 'src/**/*.js', 'src/**/*.html' ],
+        lang: ['en'],
+        dest: 'build/tmp'
```

Comment:
Ah cool, thanks @jbudz ! @hickeyma instead of `tmp` how about we call it `i18n_extract`?